aws-test/tests/aws_ecr_repository/test-list-query.sql

```diff
@@ -0,0 +1,3 @@
+select repository_name, repository_arn, image_tag_mutability, image_scanning_configuration, partition, region
+from aws.aws_ecr_repository
+where akas::text = '["{{ output.resource_aka.value }}"]'
```

Comment:
```suggestion
where akas::text = '["{{ output.resource_aka.value }}"]';
```
Please add ; for all the sql queries